### ISSUE TITLE
Redactar pequeña guía sobre como hacer preguntas en nuestros canales

### ISSUE THREAD

**Issue by stsewd** [PythonEcuador/PythonEcuador.github.io#94]
He visto muchas preguntas de muchos usari@s, la mayoría empieza con: 

- Algún experto en..
- Alguien ha usado x
- Alguien me puede ayudar con x
- Me sale un error usando x

Deberíamos escribir una pequeña guía con recomendaciones para que obtengan una respuesta más rápida, algunas cosas que se debe resaltar:

- no todos somos expertos en todas las tecnologías, pero de seguro alguien puede ayudar
- no necesariamente hay que haber usado x para resolver un problema (puede ser un error general de python, lógica, etc)
- Todos están dispuestos a ayudarte (probablemente recibas un Si! o nadie responda, porque están ocupad@s o no llama mucho la atención responder a esa pregunta). Recibirás más respuestas si describes el problema que estás teniendo.
- Asegúrate de poner un poco de contexto de lo que ya hayas hecho, el error de msg, porción de código (no imagen, tampoco pegues todo el archivo en el canal, es mejor usar una plataforma como gist)
- Recordar que todos somos voluntarios, siempre preguntar con respeto y paciencia
- No pedir que te resuelvan todo (deberes, por ejemplo)
- Y otras cosas más

**Comment by gomezgleonardob** [comment]
Tomo esta tarea ,estoy finalizando la Guía también creo un enlace desde el index

**Comment by OrpheonZeroh** [comment]
Quisiera hacer alguna pregunta

**Comment by stsewd** [comment]
@OrpheonZeroh claro, lo puedes hacer aquí :) o en nuestro canal de telegram https://t.me/pythonecuador

**Comment by ghostbounty** [comment]
1. **Claridad en la Pregunta:**
   - Inicia con una descripción clara y concisa de tu problema o duda.
   - Evita preguntar si hay expertos o si alguien ha utilizado cierta tecnología, en su lugar, ve directo al grano con tu pregunta.

2. **Proporciona Contexto:**
   - Explica brevemente lo que estás tratando de lograr y cualquier cosa que ya hayas intentado.
   - Si recibes un mensaje de error, incluye el texto exacto del mensaje (no una imagen).

3. **Muestra tu Trabajo:**
   - Comparte una porción relevante de tu código, asegurándote de que esté bien formateado. 
   - Utiliza servicios como [Gist](https://gist.github.com/) o [Pastebin](https://pastebin.com/) para compartir tu código si es demasiado largo.

4. **Sé Respetuoso y Paciente:**
   - Recuerda que las personas en la comunidad son voluntarios dispuestos a ayudar, pero también tienen sus propios compromisos.
   - Agradece las respuestas que recibas, incluso si no resuelven completamente tu problema.

5. **Evita Pedir Soluciones Completas:**
   - Es mejor aprender y entender cómo resolver un problema que simplemente obtener una solución.
   - Si se trata de tareas o deberes, intenta resolverlo por tu cuenta primero y pregunta sobre aspectos específicos donde necesites claridad.

6. **Fomenta la Colaboración:**
   - Si alguien te ayuda a resolver tu problema, considera compartir tu solución final para beneficio de otros.
   - Participa respondiendo preguntas de otros si tienes conocimiento sobre el tema.

7. **Mantenlo Técnico y al Grano:**
   - Evita desviarte del tema principal o involucrar aspectos no técnicos en la discusión.

8. **Investiga Antes de Preguntar:**
   - Haz una búsqueda rápida para ver si tu pregunta ya ha sido respondida anteriormente en la comunidad o en otros foros.

Siguiendo estas recomendaciones, estarás contribuyendo a mantener una comunidad saludable y activa, además de maximizar tus posibilidades de obtener ayuda efectiva y oportuna.
